docs/concepts/what-is-a-run-id.md

```diff
@@ -16,3 +16,11 @@ For example, a Temporal Cron Job is a chain of Workflow Executions that all have
 Each Workflow Execution within the chain is considered a _Run_.
 
 A Run Id uniquely identifies a Workflow Execution even if it shares a Workflow Id with other Workflow Executions.
+
+:::note
```

Comment:
Because of the importance of this admonition, I'm tempted to raise it from `note` to `caution`. Whaddiya think?